content/terms/reference/filters.md

```diff
@@ -4,51 +4,76 @@ title: "Filters"
 
 # Filters
 
-Some documents require more complex filtering beyond basic element selection and removal. For example, web pages often contain dynamically generated content like tracking IDs in URLs that change on each page load. While these elements are part of the page, they are not meaningful to the terms content itself. If such dynamic content is included in the archived versions, it creates a lot of insignificant versions and pollutes the archive with noise that makes it harder to identify actual changes to the terms.
-
-Filters address this need by providing a way to programmatically clean up and normalize the content before archiving. They are implemented as JavaScript functions that can manipulate the downloaded web page using the [DOM API](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model), allowing for sophisticated content transformations beyond what's possible with simple CSS selectors.
-
-Filters take the document DOM and the terms declaration as parameters and are:
+Filters are JavaScript functions that take the document DOM as parameter and are:
 
 - **in-place**: they modify the document structure and content directly;
 - **idempotent**: they return the same document structure and content even if run repeatedly on their own result.
 
-Filters are loaded automatically from files named after the service they operate on. For example, filters for the Meetup service, which is declared in `declarations/Meetup.json`, are loaded from `declarations/Meetup.filters.js`.
-
 The generic function signature for a filter is:
 
 ```js
-export [async] function filterName(document, documentDeclaration)
+export [async] function filterName(document, [parameters])
```

Comment:
It's not `filterName(document, parameters, documentDeclaration)` as it is written in the How to apply filters ?